Signal/src/environment/VersionMigrations.m

```diff
@@ -12,6 +12,7 @@
 #import "SignalKeyingStorage.h"
```

Comment:
FYI I'm hoping to nuke this class in favor of just the MigrationRunner at some point. The last migration we added to this class was 2.4.1 - 10 months ago.